src/gui/sdlmain.cpp

```diff
@@ -1461,9 +1453,36 @@ static void GUI_StartUp(Section * sec) {
 
 	}
 
+	// Read and setup the intial mouse capture state
+	const std::string capturemouse = section->Get_string("capturemouse");
+	if (capturemouse == "onstart") {
+		sdl.mouse.capture_choice = MouseCaptureType::OnStart;
+	} else if (capturemouse == "onclick") {
+		sdl.mouse.capture_choice = MouseCaptureType::OnClick;
+	} else if (capturemouse == "never") {
+		sdl.mouse.capture_choice = MouseCaptureType::Never;
+	} else {
+		// If an invalid value is provided, the section handler
+		// should have automatically set the default to 'onclick'
+		assert(sdl.mouse.capture_choice == MouseCaptureType::OnClick);
+	}
+	// Capture the mouse straight away ...
+	if (sdl.desktop.fullscreen || sdl.mouse.capture_choice == MouseCaptureType::OnStart) {
+		GFX_ToggleMouseCapture();
+	// Otherwise simply entire the cursor is visible
+	} else {
+		SDL_ShowCursor(1);
```

Comment:
Oh, `SDL_ShowCursor` usage. If we need this, then use `SDL_ShowCursor(SDL_ENABLE)`.

Reply:
Another nice improvement to make the intent clear. 
Updated.